public/docs/_examples/server-communication/dart/lib/wiki/wikipedia_service.dart

```diff
@@ -0,0 +1,24 @@
+// #docregion
+import 'dart:async';
+import 'package:angular2/angular2.dart';
+import 'package:jsonpadding/jsonpadding.dart';
+
+@Injectable()
+class WikipediaService {
+  Future<List<String>> search(String term) async {
+    // #docregion call-jsonp
+    Uri uri = new Uri(
+        scheme: 'http',
+        host: 'en.wikipedia.org',
+        path: 'w/api.php',
+        queryParameters: {
+          'search': term,
+          'action': 'opensearch',
+          'format': 'json'
+        });
+    // TODO: Error handling
+    List result = await jsonp(uri);
```

Comment:
It is the same here https://github.com/angular/angular.io/blob/master/public/docs/_examples/server-communication/ts/app/wiki/wikipedia.service.ts#L25
because the request returns more data than what we need.
Do you have any sugestion ?